_posts/2019-08-06-pre-post-events-sustainable-trains.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Pre-/Post-Events: sustainable train travels from and to RustFest Barcelona"
+authors:
+  - dns2utf8
+socialImageSrc: "/assets/posts/barcelona/cfp-opens.png"
+---
+
+## The big plan 🚆
+
+What if you could meet with other Rustaceans while you travel to and from RustFest Barcelona?
+
+So what do we need for that?
+
+1. Is there a path to go by train?
+2. How far can you go in one day?
+3. Which stops are interesting?
+
+### Tickets or Interrail
+
+Usually people buy tickets from A to B, sometimes by locking in on a single connection.
+However, a travel of this size is different.
+
+When traveling huge distances over many countries sometimes delays happen or a train might break down.
+With a regular ticket you have to find staff and organize a reroute on the spot.
+
+This is where Interrail saves the day.
+With the Interrail global pass you buy travel days over a period of time.
+So you can simply catch the next train and continue your journey.
+
+Eg. buy 7 travel days within one months.
```

Comment:
This is an example sentence as a standalone one.

I'd merge it with the next sentence:

```suggestion
If you buy 7 travel days within one months, you have the flexibility to ride any train outside of your home country on 7 days.
```

Reply:
Yes, this sounds better